src/components/pages/edit-doc.js

```diff
@@ -54,13 +54,9 @@ class EditDoc extends React.Component {
 
     autoBind(this);
 
-    const { editorFactory, rendererFactory, documentApiClient, initialState } = this.props;
+    const { initialState } = this.props;
     const { documentRevision, proposedSections } = initialState;
 
-    this.editorFactory = editorFactory;
```

Comment:
Why are these changes here? It doesn't seem to be part of the story.

Reply:
Linting rules have changed and it now realizes that this line is not used